libpod/rootless_cni_linux.go

```diff
@@ -22,10 +23,9 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// Built from ../contrib/rootless-cni-infra.
 var rootlessCNIInfraImage = map[string]string{
-	// Built from ../contrib/rootless-cni-infra
-	// TODO: move to Podman's official quay
-	"amd64": "ghcr.io/akihirosuda/podman-rootless-cni-infra:gd34868a13-amd64",
+	"amd64": "quay.io/libpod/rootless-cni-infra@sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4", // 1-amd64
```

Comment:
Not pushed?

```console
$ podman pull quay.io/libpod/rootless-cni-infra@sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4
Trying to pull quay.io/libpod/rootless-cni-infra@sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4...
  manifest unknown: manifest unknown
Error: unable to pull quay.io/libpod/rootless-cni-infra@sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4: Error initializing source docker://quay.io/libpod/rootless-cni-infra@sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4: Error reading manifest sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4 in quay.io/libpod/rootless-cni-infra: manifest unknown: manifest unknown
```

Reply:
I copied the sha from quay yesterday but it may was out of date. Rebuilding now :+1: